### ISSUE TITLE
remove jquery dependancy from key_tag button insertion

### ISSUE THREAD

**Issue by urandom2** [grdme/grd.me#2]
@winhowes says that using jquery here is bad pratice because "there could be a parent element above the button which won't let the event bubble". so we should use `addEventListener` instead.

**Comment by winhowes** [comment]
Yeah I want to see if doing `addEventListener` with capturing set to `true` would get around that issue. Otherwise - we may want to looking into attaching the listener to the actual element instead of the `body` tag (again with capturing set to `true`). The downside here would be performance as adding listeners is expensive
